paasta_tools/cli/cmds/status.py

```diff
@@ -301,9 +305,11 @@ def paasta_status_on_api_endpoint(
         output.append(str(e))
         return 1
 
-    if status.git_sha != "":
+    if status.version and status.version != "":
+        output.append("    Version:    %s (desired)" % status.version)
```

Comment:
super nit, it'd be probably better to start using f-strings like in other places